Story/DarkDungeon.cs

```diff
@@ -0,0 +1,61 @@
+/*
+name: DarkDungeon
+description: This script finish all dark dungeon quests storyline in /darkdungeon
+tags: dark, dungeon, story, line, mirror, drakath
+*/
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+using Skua.Core.Interfaces;
+
+public class DarkDungeon
+{
+    private IScriptInterface Bot => IScriptInterface.Instance;
+    private CoreBots Core => CoreBots.Instance;
+    private CoreStory Story = new();
+
+
+    public void ScriptMain(IScriptInterface Bot)
+    {
+        Core.SetOptions();
+
+        storyline();
+        Core.SetOptions(false);
+    }
+
+    public void storyline()
+    {
+        if (Core.isCompletedBefore(3545))
+            return;
+
+        Story.PreLoad(this);
+        //Bright Idea 2909
+        Story.MapItemQuest(2909, "battleoff", 17375);
+
+        //Spare Parts 2910
+        Story.MapItemQuest(2910, "battleoff", 1780, 8);
+
+        //Power It Up 2911
+        Story.KillQuest(2911, "battleoff", "Evil Moglin");
+
+        //Filthy Creatures 2912
+        Story.KillQuest(2912, "battleoff", "Evil Moglin");
```

Comment:
These quests are not a part of this storyline, this is battleoff, not darkdungeon. The script for battleoff already exists.